### DISCUSSION TITLE
空 routes，真的有必要存在吗？

### DISCUSSION THREAD

**Discussion by ppsite** [umijs/umi#7057] [General]
## 背景：

准备用 umijs 写个 utools 插件，需要所有资源本地加载；`umi build` 后直接打开 index.html 出现白屏；查阅 umi 文档后提示说：

> 相对路径 ./ 有一些限制，例如不支持多层路由 /foo/bar，只支持单层路径 /foo

## 最终解决办法:

```
# .umirc.ts

import { defineConfig } from 'umi';

export default defineConfig({
  nodeModulesTransform: {
    type: 'none',
  },
  publicPath: './',   # 添加一行
  routes: [{ path: '/', component: '@/pages/index', routes: [] }],     # 添加 routes:[]
  fastRefresh: {},
});
```

## 问题点：

经测试，不添加 routes:[] 仍然会显示白屏，有没有什么办法，把这个多此一举的玩意儿省略掉，害死强迫症！！！

**Comment by xiaohuoni** [comment]
你把整个 routes 配置删掉，应该就可以了
